docs/root/configuration/listeners/network_filters/_include/wasm-filter-proxy.yaml

```diff
@@ -0,0 +1,25 @@
+static_resources:
+  listeners:
+  - name: main
+    address:
+      socket_address:
+        address: 127.0.0.1
+        port_value: 10000
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.wasm
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.wasm.v3.Wasm
```

Comment:
@hanxiaop not 100% sure what the issue is here - this is the error:

```console
C++ exception with description "Function call: customStatNamespaces
    The mock function has no default action set, and its return type has no default value set." thrown in the test body.
```

i think you probably beed to add a `stat_prefix` for this filter also - altho checking the existing wasm config example for http.wasm it doesnt have it so not sure (`examples/wasm-cc/envoy.yaml`) - defo worth a try

Reply:
can you start an Envoy server with this config + a wasm bin ?

Reply:
> can you start an Envoy server with this config + a wasm bin ?

@phlax Yes I do.

At least it's not `no directory found` for the wasm bin. I'll try different configs out, thanks for the help!